src/screens/Followings/FollowersScreen.js

```diff
@@ -12,14 +13,17 @@
 import {fonts} from '../../utils/fonts';
 
 function FollowersScreen() {
+  const route = useRoute();
+  const {params} = route;
   const navigation = useNavigation();
   const [profileState, dispatchNavbar] = React.useContext(Context).profile;
 
   const isAndroid = Platform.OS === 'android';
 
   const [isLoading, setIsLoading] = React.useState(false);
-  const [dataFollower, setDataFollower] = React.useState([]);
+  const [dataFollower, setDataFollower] = React.useState(params?.params?.initialData || []);
   const [searchText, setSearchText] = React.useState('');
+  const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of `route`, `params`, and `isFirstTimeOpen` are significant changes. Make sure to handle these new variables correctly throughout the component. Also, ensure that `params?.params?.initialData` is always provided with the correct data structure to prevent runtime errors.
```diff
+ const route = useRoute();
+ const {params} = route;
+ const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->